internal/pkg/build/assemblers/assembler_sif.go

```diff
@@ -177,7 +185,28 @@ func (a *SIFAssembler) Assemble(b *types.Bundle, path string) (err error) {
 		return fmt.Errorf("While running mksquashfs: %v: %s", err, strings.Replace(string(errOut), "\n", " ", -1))
 	}
 
-	err = createSIF(path, b.Recipe.Raw, b.JSONObjects["oci-config"], squashfsPath)
+	if b.Opts.Encrypted {
+
```

Comment:
And here? Are these necessary?

Reply:
Not necessary. Coming from C background, I am used to use new lines liberally for readability. Will remove